testing/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//testing:bmv2_parsing_check.bzl", "subset_diff_test")
+
+test_suite(
+    name = "parsing",
+    tests = [
+      ":table_test",
+      ":basic_test"
+    ]
+)
+
+subset_diff_test(
```

Comment:
One big advantage of simple diff testing is that you get to see the output, this is lost with this definition of subset_diff_test. That is, I can't see in this review what the protobuf looks like for the given P4 program, which seems like a big downside.

Could we make it so that the protobuf is the expected output and is what is checked in?

Reply:
I am going to check in the output of the protobuf parsing then dumping, we can use this for golden files testing as well as the subset diff testing